src/Apps/Artist/Routes/AuctionResults/TableSidebar.tsx

```diff
@@ -26,12 +26,14 @@ interface Props {
 
 export const TableSidebar = (props: Props) => {
   return (
-    <Responsive>
-      {({ xs }) => {
-        if (xs) return <SmallTableSidebar {...props} />
-        else return <LargeTableSidebar {...props} />
-      }}
-    </Responsive>
+    <>
+      <Media at="xs">
+        <SmallTableSidebar {...props} />
+      </Media>
+      <Media greaterThan="xs">
```

Comment:
I really, really like this api. It looks better, reads easier, and feels cleaner than the switch above.